README.md

```diff
@@ -0,0 +1,96 @@
+https://api.nasa.gov/
+
+The APOD app shows a collection of the last 30 images provided by the NASA api. Everyday, at midnight USA eastern standard time (USA East Coast time), a new pictured is added to the collection.
+
+The pictures rendered by the app include images and videos. The app provides the user with a select option, to show a separated list of images or pictures. By default, the app shows the combined list of images and videos, under de option "Both".
+
+Under each image or video displayed there's a button that let the user navigate to a new section containing the complete information provided by the api about the selected image or video.
+
+This information includes the following items:
+- Title
+- Author (if available)
+- Date published (YYYY-MM-DD format)
+- Explanation
+- Media service
+- Service code
+
+* Note on the size of images in the details section: for educational and entertaiment porpouses, the size of the image in this section has been set to a very big size, which lets the user to explore small sections of each image with the scroll of the browser. 
+
+The APOD app has been built with React and Redux / Redux Toolkit.
+
+## Requirements:
+
+Project will be build according to the following guidelines:
+
+Use React documentation.
+Use React components.
+Use React props.
+Use React Router.
+Connect React and Redux.
+Handle events in a React app.
+Write unit tests with React Testing Library.
+Use styles in a React app.
+Use React life cycle methods.
+Apply React best practices and language style guides in code.
+Use store, actions and reducers in React.
+
+## Built With:
+Major languages: Javascript, HTML, CSS
+Frameworks: webpack, Jest, React, Redux, Redux Toolkit
+Technologies used: VSC, Github, Gitflow
+
+## Getting Started:
+Clone the repository in your local machine using git clone.
+
+Please run: npm run build and then npm start to see project displayed in your browser.
+
+## Live demo:
+
+To see the project deployed please visit anyone of the following links:
+https://tranquil-fudge-35218d.netlify.app/
+
+To see the videp presentation of the app, please visit the following link:
+https://www.loom.com/share/b1332bbcb84d4477924818c1f41a9834
+
+
+##  Authors
+👤 Author1
+
+- Dario Alessio
+
+- GitHub: https://github.com/DarioAlessioR
+
+- Slack: https://microverse-students.slack.com/team/U039GCFRK9B
+
+- LinkedIn: https://www.linkedin.com/in/dario-alessio-3a3b7911b
+
+
+## 🤝 Contributing:
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support:
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To the NASA for providing the images and videos, as well as the scientific information of each picture.
+- To Nelson Sakwa: the author of the original design by the Creative Commons License.
+- Hat tip to anyone whose code was used
+- Inspiration
```

Comment:
- [ ] [REQUIRED] Good job so far! You did a good job on this part, but you missed a few `README` requirements.
- At `line 90`, you should have written the requirements mentioned by the [previous code reviewer](https://github.com/DarioAlessioR/capstone-apod/pull/4#discussion_r943070735) in `link format`(EG: `[NAME THE LINK](PASTE THE LINK)`, but you wrote it in `text format`. This may cause unknown questions when you look at the `README` description of any recruiter project in the future. So please fulfill my small and easy requests. 👍
 
- [ ] [HINT] It should look like this =>

" [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). "

Reply:
Change done. Thank you @mukhammadrizooff